### ISSUE TITLE
Add on-demand metric calculations

### ISSUE THREAD

**Issue by sgvictorino** [sgvictorino/linear-regression-cmd#3]
**Scenario**
Right now there's no way to calculate metrics after initially training a model.

**Proposed Solution**
Add a command that can calculate metrics on an existing model file.

**Steps to Closure**
This issue should be closed when there's a way to retrieve metrics after running `--train` to serialize a model to the filesystem.

**Does this propose a breaking change?**
It's dependent on #1, so yes.

**Is this dependent on another issue?**
#1 has to be resolved first because the input data needs to be stored in the file to pass to `cross_validate` on-demand.
